### PR TITLE
Move constants away from common package

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area dev-productivity
/kind cleanup
/priority normal

**What this PR does / why we need it**:
This PR is a good step towards eliminating the common package (or, at least, to better declare its purpose). It moves some constants to better places.
Also, constants for the image names in the image vector are now auto-generated to prevent double maintenance and ambiguous naming.

/squash

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: hack/generate-imagename-constants.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+const ("
+
+for image_name in $(yaml2json < "$PATH_CHARTS/images.yaml" | jq -r '[.images[].name] | unique | .[]'); do
+  variable_name="$(camelCase "$image_name")"
+
+  out="
+$out
+	// ImageName$variable_name is a constant for an image in the image vector with name '$image_name'.
+	ImageName$variable_name = \"$image_name\""
```
Comment (timebertt) [inline hack/generate-imagename-constants.sh:42]: Very nice, I like it 🚀

File: hack/generate-imagename-constants.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+done
+
+out="
+$out
+)
+"
+
+echo "$out" > "$PATH_CHARTS/images.go"
+go fmt "$PATH_CHARTS/images.go"
```
Comment (timebertt) [inline hack/generate-imagename-constants.sh]: Use `goimports` instead to have it consistent with `make format`?

**Review by timebertt — DISMISSED** [review]
Very nice PR, thanks for cleaning up!
/lgtm

File: hack/generate-imagename-constants.sh (hunk 2 above)
Comment (rfranzke) [reply hack/generate-imagename-constants.sh]: Good point, done

**Review by timebertt — APPROVED** [review]
Thanks!
/lgtm

**Pull request merged**